app/src/main/java/com/wizeline/mobilenews/domain/models/NewsArticle.kt

```diff
@@ -4,13 +4,13 @@ import com.google.gson.annotations.SerializedName
 import java.io.Serializable
 
 data class NewsArticle(
-    val title: String,
+    val title: String?,
     val author: String?,
     @SerializedName("published_date")
-    val publishedDate: String,
-    val link: String,
-    val excerpt: String,
-    val summary: String,
+    val publishedDate: String?,
+    val link: String?,
+    val excerpt: String?,
+    val summary: String?,
```

Comment:
Are these changes necessary? The Newscather service does not return null values inside an article, but I do not not if the pager library requires nullable parameters when you do the render in the view.